### ISSUE TITLE
StorybookJS

### ISSUE THREAD

**Issue by rickhumphries** [buttercupstraining/lrnwebcomponents#10]
I really like the [Storybook](https://storybook.js.org/docs/web-components/get-started/introduction) setup PSU have:
https://webcomponents.psu.edu/styleguide/?path=/story/about-getting-started--using-penn-state-cdn

Would be great to have this as a place for us to document our components and have a live style guide

**Comment by JackWillDavis** [comment]
Yep, I'm using [WCF](https://github.com/elmsln/WCFactory) currently for generating our elements so in the end we will have a structure similar to the lrnwebcomponents repo. Storybook is built in to that and I'm able to use it for setting up demo/style guides and view elements locally until they're ready to join ELMSLN.

Happy to use another library if you weren't keen on WCF your end :)
